src/components/types.ts

```diff
@@ -0,0 +1,14 @@
+interface IRowAndColumn {
```

Comment:
build fails as it's not an exported type

Reply:
also, move the file to a folder outside `components` into something like `shared/types.ts`